config/base/params.env

```diff
@@ -1,2 +1 @@
-monitoring-namespace=opendatahub
 odh-model-controller=quay.io/opendatahub/odh-model-controller:v0.11.1.1
```

Comment:
This is different from the original PR https://github.com/opendatahub-io/odh-model-controller/pull/162